src/entities/models/dto.py

```diff
@@ -33,6 +33,16 @@ class FilingTaskStateDTO(BaseModel):
     change_timestamp: datetime | None = None
 
 
+class ContactInfoDTO(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    id: int | None = None
+    first_name: str
+    last_name: str
+    email: str
+    phone: str
+
```

Comment:
Please add address fields as well. @billhimmelsbach will send a screenshot of the designs for address fields.